__tests__/index.test.ts

```diff
@@ -92,16 +92,16 @@ describe('cli', () => {
     await expect(cli([])).resolves.toContain('OpenAPI / Swagger');
   });
 
-  describe('stored API key', () => {
+  describe('stored API key via configstore', () => {
```

Comment:
Want to nest this and "stored API key via env vars" under a single "stored API key" `describe`?